viewing_party/party.py

```diff
@@ -1,23 +1,144 @@
 # ------------- WAVE 1 --------------------
 
+from cmath import rect
+from time import monotonic
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+
+    if title and genre and rating:
+        movie_dict['title'] = title 
+        movie_dict['genre'] = genre
+        movie_dict['rating'] = rating
+    else:
+        return None
+    return movie_dict

```

Comment:
I generally find it clearer to return a dictionary literal instead of creating an empty dictionary and modifying it:

```suggestion
    if title and genre and rating:
        return {
            'title': title,
            'genre': genre,
            'rating': rating
        }
    else:
        return None
```